core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -609,26 +609,27 @@ static bool core_mmu_entry_copy(struct core_mmu_table_info *tbl_info,
 }
 
 static void core_init_mmu_prtn_tee(struct mmu_partition *prtn,
-				   struct tee_mmap_region *mm)
+				   struct memory_map *mem_map)
 {
-	size_t n;
+	size_t n = 0;
 
-	assert(prtn && mm);
+	assert(prtn && mem_map);
 
-	for (n = 0; !core_mmap_is_end_of_table(mm + n); n++) {
```

Comment:
`core_mmap_is_end_of_table()` is now unused and should be removed from `core/include/mm/core_mmu.h`, and with that `MEM_AREA_END` should be removed from `enum teecore_memtypes` as well.

Reply:
Good point, I'll update.